clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -26,6 +26,14 @@ class CIR_Attr<string name, string attrMnemonic, list<Trait> traits = []>
   let mnemonic = attrMnemonic;
 }
 
+class UnitAttr<string name, string attrMnemonic, list<Trait> traits = []>
```

Comment:
Since this is a class, not an actual definition, we should probably rename it to `CIRUnitAttr`, just to make it clear this is CIR's custom version of it.